src/layout/Foot.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Card, CardBody, Bullseye, Grid, GridItem, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import { BookIcon, OutlinedEnvelopeIcon, ChatIcon } from '@patternfly/react-icons';
+
+const Foot = () => (
+    <Card>
```

Comment:
Looking at the mocks, this seems to have a max width of like `80%` and then center the content

Reply:
Ooh, nice catch. I was wondering how to style it to center and this one does the trick. I have set it up that on large screens (>992px) it will be affected by the 80% rule, but on smaller screens it's not.